### PR TITLE
DOCA 2.6 from public repo

### PR DESCRIPTION
**Pull request by e-ago**
## Description
Modified the Dockerfile if DOCA is enabled to install with apt DOCA debian packages from public repo instead of the internal repo

## By Submitting this PR I confirm:
- I am familiar with the [Contributing Guidelines](https://github.com/nv-morpheus/Morpheus/blob/main/docs/source/developer_guide/contributing.md).
- When the PR is ready for review, new or existing tests cover these changes.
- When the PR is ready for review, the documentation is up to date with these changes.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -149,12 +149,76 @@ FROM conda_env as base_extended
...
+        openvswitch-switch \
+        srptools \
+        mlnx-ethtool \
+        mlnx-iproute2 \
+        dpcp \
+        openmpi \
+        ucx \
+        doca-prime-runtime \
+        doca-prime-sdk \
+        #doca-runtime \
```
Comment (mdemoret-nv) [inline docker/Dockerfile:208]: Should this be removed?

File: docker/Dockerfile [hunk 2]
```diff
@@ -149,12 +149,76 @@ FROM conda_env as base_extended
...
+        doca-sdk \
+        doca-gpu \
```
Comment (mdemoret-nv) [inline docker/Dockerfile:210]: Can we alphabetize this list?

File: docker/Dockerfile [hunk 3]
```diff
@@ -149,12 +149,76 @@ FROM conda_env as base_extended
...
+        libibmad5 \
+        libopenvswitch \
+        openvswitch-common \
+        openvswitch-switch \
+        srptools \
+        mlnx-ethtool \
+        mlnx-iproute2 \
+        dpcp \
+        openmpi \
+        ucx \
```
Comment (mdemoret-nv) [inline docker/Dockerfile:205]: Can we move this into the `doca.sh` to be contained with the other DOCA install steps?

**Review by mdemoret-nv — CHANGES_REQUESTED** [review]

**Comment by e-ago** [comment]
@mdemoret-nv I've applied your comments

**Review by mdemoret-nv — APPROVED** [review]

**Review by mdemoret-nv — APPROVED** [review]
Have built the new release and dev container with DOCA support. Release container has been tested by @e-ago. Approved.

**Pull request merged**